packages/api/src/cli/commands/install.ts

```diff
@@ -26,7 +26,8 @@ cmd
       'ts',
     ])
   )
-  .action(async (uri: string, options: { lang: string }) => {
+  .addOption(new Option('-i, --install', 'Do not prompt to install packages'))
```

Comment:
I agree. I also agree that the option could use a better name, but when thinking of it, I also came up empty. It could be something like `-s, --skip-install-prompt` ? I think it feels a bit long but it is definitely more understandable.

Reply:
Let's do `-y, --yes` as that matches some preexisting work in commands like `npm init`:

```
   yes

       • Default: null

       • Type: null or Boolean

       Automatically answer "yes" to any prompts that npm might print on the command line.
```

Reply:
Updated. We could also do `--noninteractive` but I am content with `--yes`